src/api/che.ts

```diff
@@ -197,13 +199,37 @@ export class CheHelper {
     }
   }
 
-  async createWorkspaceFromDevfile(namespace: string | undefined, devfilePath = '', workspaceName: string | undefined, accessToken = ''): Promise<string> {
+  async startWorkspace(cheNamespace: string, workspaceId: string, accessToken?: string): Promise<void> {
+    const cheUrl = await this.cheURL(cheNamespace)
+    const endpoint = `${cheUrl}/api/workspace/${workspaceId}/runtime`
+    let response
+
+    const headers: {[key: string]: string} = {}
+    if (accessToken && accessToken.length > 0) {
+      headers.Authorization = accessToken
+    }
+    try {
+      response = await this.axios.post(endpoint, undefined, { headers })
+    } catch (error) {
+      if (error.response && error.response.status === 404) {
```

Comment:
`getCheApiError` function actually catches  404 error (rebase on master pls)

Reply:
Not only...

Reply:
I want to handle 404 error in another way for this specific request.